src/libs/OptionsListUtils.js

```diff
@@ -587,6 +587,138 @@ function isCurrentUser(userDetails) {
     return _.some(_.keys(loginList), (login) => login.toLowerCase() === userDetailsLogin.toLowerCase());
 }
 
+/**
+ * Build the options for the tree hierarchy via indents
```

Comment:
```suggestion
 * Build the options for the category tree hierarchy via indents
```

NAB: just a little bit more specific since I think this is only used for categories.